packages/ihe-gateway-sdk/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metriport/ihe-gateway-sdk",
+  "version": "0.1.0",
+  "description": "SDK to interact with other IHE Gateways - by Metriport Inc.",
+  "author": "Metriport Inc. <contact@metriport.com>",
+  "homepage": "https://metriport.com/",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js",
+    "./client/*": "./dist/client/*.js",
+    "./models/*": "./dist/models/*.js",
+    "./common/*": "./dist/common/*.js"
+  },
+  "typesVersions": {
+    "*": {
+      "/": [
+        "dist/index.d.ts"
+      ],
+      "client/*": [
+        "dist/client/*"
+      ],
+      "models/*": [
+        "dist/models/*"
+      ],
+      "common/*": [
+        "dist/common/*"
+      ]
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE",
+    "package.json"
+  ],
+  "repository": {
+    "url": "https://github.com/metriport/metriport.git",
+    "type": "git",
+    "directory": "packages/ihe-gateway-sdk"
+  },
+  "scripts": {
+    "clean": "rimraf dist && rimraf node_modules",
+    "tsc": "tsc",
+    "watch": "tsc --watch",
+    "build": "tsc -p .",
+    "build:cloud": "npm run build",
+    "typecheck": "tsc --noEmit",
+    "lint": "npx eslint . --ext .ts",
+    "lint-fix": "npm run lint --fix",
+    "prettier-fix": "npx prettier '**/*.ts' --write",
+    "test": "jest --runInBand --detectOpenHandles --passWithNoTests",
+    "test:e2e": "E2E=true jest --runInBand --detectOpenHandles --passWithNoTests"
+  },
+  "bugs": {
+    "url": "https://github.com/metriport/metriport/issues"
+  },
+  "dependencies": {
+    "axios": "^1.3.5",
+    "zod": "^3.22.1"
+  }
```

Comment:
missing dependency to `shared`

Reply:
once i include it, i start getting that "Cannot set properties of null (setting 'dev')"..

Reply:
I checked out this branch, added it like below (manually on the package.json file, not through `npm i`) and it works fine:

`package.json`

<img width="520" alt="image" src="https://github.com/metriport/metriport/assets/2132564/d7574de0-a2be-4bac-8fe0-02c5508673af">

then I `npm i` so it updates the lock file:

`package-lock.json`

<img width="1419" alt="image" src="https://github.com/metriport/metriport/assets/2132564/87275258-bda7-4f15-b2ac-739d2bedce62">

<img width="1373" alt="image" src="https://github.com/metriport/metriport/assets/2132564/64d953cc-597c-42c2-bf1f-97e810cad924">

`npm i`

<img width="519" alt="image" src="https://github.com/metriport/metriport/assets/2132564/18c600b7-849c-4950-b344-4dad496ea61b">

`npm run build`

<img width="432" alt="image" src="https://github.com/metriport/metriport/assets/2132564/a3957adf-2e02-482b-bc4e-5f405eae42cf">

Reply:
if it keeps failing we can huddle